config/default/testRunner.conf.php

```diff
@@ -502,6 +502,11 @@
      * This is required for caching scenarios
      * @type integer
      */
-    'item-cache-size' => 3
+    'item-cache-size' => 3,
+
+    /**
+     * Keep timer for not allowed answer
+     */
+    'keep-timer-notallowed' => false
```

Comment:
I don't think you need a flag for that. The behavior should be the same for the whole platform. 
Since the item is displayed in the behind, the timer should continue to countdown